applications/RomApplication/python_scripts/structural_mechanics_dynamic_implicit_rom_solver.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Import applications
+import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
+
+# Import base class file
+from KratosMultiphysics.StructuralMechanicsApplication.structural_mechanics_implicit_dynamic_solver import ImplicitMechanicalSolver
+import KratosMultiphysics.RomApplication as romapp
+
+def CreateSolver(main_model_part, custom_settings):
+    return ROMSolver(main_model_part, custom_settings)
+
+class ROMSolver(ImplicitMechanicalSolver):
+    """The stationary class for ROM structural mechanics solvers.
+
+    Public member variables:
+    stationary_settings -- settings for the implicit dynamic solvers.
+
+    See structural_mechanics_implicit_dynamic_solver.py for more information.
+    """
+
+    def __init__(self, main_model_part, custom_settings):
+        super(ROMSolver, self).__init__(main_model_part, custom_settings)
+        KratosMultiphysics.Logger.PrintInfo("::[ROMSolver]:: ", "Construction finished")
+
+    #### Private functions ####
+    @classmethod
+    def GetDefaultSettings(cls):
+        default_settings = KratosMultiphysics.Parameters("""
+        {
+            "rom_settings": {
+            "nodal_unknowns": [ "DISPLACEMENT_X", "DISPLACEMENT_Y", "DISPLACEMENT_Z"],
```

Comment:
just to be sure: this list is still needed?

if not: why it is not validated?

Reply:
In this PR, these parameters are required; however they can be inferred from the model part we passed to the solver. Therefore,  #6325 modifyes theB&S so that this parameters are not  entered anymore.